guides/common/modules/proc_distributing-archived-content-view-versions.adoc

```diff
@@ -0,0 +1,22 @@
+[id="Distributing_Archived_Content_View_Versions_{context}"]
+= Distributing Archived Content View Versions
+
+The setting **Distribute archived content view versions** enables hosting of non-promoted Content View version repositories in the {Project} content web application along with other repositories.
+This is useful while debugging to see what content is present in your Content View versions.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Administer* > *Settings*.
+. Click the *Content* tab.
+. Set the *Distribute archived content view versions* parameter to *Yes*.
+. Click *Submit*.
++
+This enables the repositories of Content View versions without lifecycle environments to be distributed at `{foreman-example-com}/pulp/content/_My_Organization_/content_views/_My_Content_View_/_My_Content_View_Version_/`.
++
+[NOTE]
+====
+Older non-promoted Content View versions are not distributed once the setting is enabled.
+Only new Content View versions become distributed.
+====
+To see the published Content View version, do the following:
```

Comment:
I'm not sure if this is needed since the version itself in the UI won't change in appearance.